game-core/build.gradle

```diff
@@ -145,15 +147,16 @@ task cleanRemoteLibs(type: Delete, group: LifecycleBasePlugin.BUILD_GROUP, descr
     delete remoteLibsDir
 }
 
-task integTest(type: JavaExec, dependsOn: [compileIntegTestJava]) {
+test {
+    useJUnitPlatform()
```

Comment:
If the normal tests are being migrated as well, there's no longer a need for the gradle junit plugin that we depend on above

Reply:
The custom jacoco would need to go too, that and the plugin were removed  in: fcbb37e